### ISSUE TITLE
How to follow someone else's follow up?

### ISSUE THREAD

**Issue by huseyinkarael** [DorsetDevStudio/Instagram_Bot#12]
How to follow someone else's follow up?

**Comment by pbh-stu** [comment]
I'll add this feature tonight.

**Comment by huseyinkarael** [comment]
Can we use many vehicles at the same time? for example to follow and like.

**Comment by pbh-stu** [comment]
yes, you should be able to run the bot multiple times on the same machine but i have not tested it.

**Comment by huseyinkarael** [comment]
it would be better if they did something else on each tab.

**Comment by pbh-stu** [comment]
good idea, I'll look at having multiple tabs running, so one liking, another following, another commenting etc..., as chrome shares sessions across tabs it would have to be the same instagram account but in theory it can launch multiple chrome browsers each working on a different instagram account.

I'll test this out later this evening and hopefully be able to publish an update within the next 12-24 hours.

**Comment by huseyinkarael** [comment]
It's a good idea to open a tab to do many things at the same time.
